dependencies/common/install-sccache

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+#
+# install-sccache
+#
+# Copyright (C) 2022 by RStudio, PBC
```

Comment:
We 
```suggestion
# Copyright (C) 2023 by RStudio, PBC
```

We'll need to go through and update our copyrights with the new year, but best not to introduce any new 2022 Copyrights.

Reply:
Updated!